test_scripts/Smoke/API/001_SetGlobalProperties_PositiveCase_SUCCESS.lua

```diff
@@ -0,0 +1,145 @@
+---------------------------------------------------------------------------------------------------
+-- User story: Smoke
+-- Use case: SetGlobalProperties
+-- Item: Happy path
+--
+-- Requirement summary:
+-- [SetGlobalProperties] SUCCESS on TTS.SetGlobalProperties and UI.SetGlobalProperties
+--
+-- Description:
+-- In case mobile application sends valid SetGlobalproperties_request with "timeoutPrompt"
+-- and/or "helpPrompt" and at least one other valid parameter, SDL must transfer from mobile
+-- app to HMI the both UI.SetGlobalProperties and TTS.SetGlobalProperties. On getting "SUCCESS"
+-- result code from the both HMI-portions, SDL must transfer "resutCode:SUCCESS", success:"true"
+-- to mobile application.
+
+-- Pre-conditions:
+-- a. HMI and SDL are started
+-- b. appID is registered and on SDL
+-- c. appID is in Background, Full or Limited HMI level
+
+-- Steps:
+-- appID requests SetGlobalproperties with timeoutPrompt, helpPrompt and other valid parameters
+
+-- Expected:
+
+-- SDL validates parameters of the request
+-- SDL checks if UI interface is available on HMI
+-- SDL checks if TTS interface is available on HMI
+-- SDL checks if SetGlobalproperties is allowed by Policies
+-- SDL checks if all parameters is allowed by Policies
```

Comment:
@AKalinich-Luxoft parameters **are**

Reply:
@AByzhynar fixed in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/1697/commits/1f9a96a07f40b1051bd1cd0e5a3b77a979c40def